### DISCUSSION TITLE
Error pushing project folder to remote repository.

### DISCUSSION THREAD

**Discussion by shyo21** [ANLAB-KAIST/KENSv3#131] [Project Q&A 1. Application Layer (Socket)] [answered]
![image](https://github.com/ANLAB-KAIST/KENSv3/assets/79900341/f4155057-1063-4735-9f61-16bae8fd16f1)

I just pushed project folder to my private repository without any modification.

It seems that pushing files is done without error, but github shows this error message.

I googled a bit but couldn't find what those message means and how to resolve it.

Thanks.

**Comment by Pusnow** [comment, marked as answer]
The checks are related to Github Actions. Please remove the `.github` directory in your private repository.
